app/lua/lnodemcu.c

```diff
@@ -118,6 +118,7 @@ LUALIB_API int luaL_posttask( lua_State* L, int prio ) {          // [-1, +0, -]
 }
 #else
 LUALIB_API int luaL_posttask( lua_State* L, int prio ) { 
+  (void)L; (void)prio;
```

Comment:
Again use `UNUSED` macro.  Actually can I make this a blanket comment relating to all `(void)var` uses in order to suppress compile warnings?  I'll drop following explicit mark-ups